index.html

```diff
@@ -34,12 +34,163 @@ <h1>I’m Omayio Glad to see you!</h1>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case4">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
       </section>
+      <section id="about-me" class="flex">
+        <h2>About Myself</h2>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h5>LET’S CONNECT</h5>
+        <ul class="flex">
+          <li><a href="#"><img src="imgs/Github.png" alt="Github Icon" /></a></li>
+          <li><a href="#"><img src="imgs/Linkedin-icon.png" alt="Linkedin-icon" /></a></li>
+          <li><a href="#"><img src="imgs/hands.png" alt="hands" /></a></li>
+          <li><a href="#"><img src="imgs/Twitter.png" alt="Twitter Icon" /></a></li>
+          <li><a href="#"><img src="imgs/M.png" alt="M icon" /></a></li>
+        </ul><br><br>
+        <button type="submit">Get My Resume</button>
+        <ul class="lang-section">
+          <li>
+            Languages
+            <img src="imgs/Icon-Down.png" alt="#" />
+          </li>
+        </ul>
+        <div class="lang-prog">
+          <img class="javascript-img" src="imgs/JS.png" alt="Javascript">
+          <span> &nbsp; &nbsp; JavaScript</span>
+        </div>
+
+        <div class="lang-prog">
+          <img class="html-img" src="imgs/HTML.png" alt="HTML">
+          <span> &nbsp; &nbsp; HTML</span>
+        </div>
+
+        <div class="lang-prog">
+          <img class="css-img" src="imgs/CSS.png" alt="css">
+          <span> &nbsp; &nbsp; JavaScript</span>
+        </div>
+
+
+        <ul class="lang-section hg">
+          <li>
+            Frameworks
+            <img src="imgs/Icon-Right.png" alt="" />
+          </li>
+          <li>
+            Skills <img src="imgs/Icon-Right.png" alt="" />
+          </li>
+        </ul>
+      </section>
+
+    </div>
+
+    <footer>
+      <h3>Contact me</h3>
+      <p>
+        If you have an application you are interested in developing, 
+        a feature that you need built or a project that needs coding. 
+        I’d love to help with it 
+      </p>
+      <form action="https://formspree.io/f/2" method="post">
```

Comment:
#### The form is not getting submitted because the action's URL is incorrect. Kindly revisit this [activity](https://formspree.io/html/) on formspree 👍🏻

Reply:
@zainsadaqat , This is now done and the form is getting submitted